README.md

```diff
@@ -102,6 +102,14 @@ A sample config file template is provided in /project/samples
 1. Enter a username and password of your Jira administrator account
 2. Provide groups to add the new user to
 
+### Google Spreadsheet
```

Comment:
I'm guessing this is not related in any way to the code changes below? It just confused me that there was something mentioning Google Spreadsheet in the README and then no code related to it in the code change. :)